components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -317,13 +318,26 @@ class BrowserToolbar @JvmOverloads constructor(
         contentDescription: String,
         visible: () -> Boolean = { true },
         @DrawableRes background: Int? = null,
+        private val padding: Padding = Padding(
+            ACTION_PADDING_DP,
+            ACTION_PADDING_DP,
+            ACTION_PADDING_DP,
+            ACTION_PADDING_DP
+        ),
         listener: () -> Unit
     ) : Toolbar.ActionButton(imageResource, contentDescription, visible, background, listener) {
         override fun createView(parent: ViewGroup): View {
             val view = super.createView(parent)
-
-            val padding = view.resources.pxToDp(ACTION_PADDING_DP)
-            view.setPadding(padding, padding, padding, padding)
+            val (start, top, end, bottom) = padding
+
+            with(view.resources) {
+                view.setPadding(
+                    pxToDp(start),
```

Comment:
The Android API is `setPadding(left, ...)` not `setPadding(start, ...)`, right? So I think you'd have to manually swap left/right to start/end based on the locale (unless this method exists already and I haven't seen it).

Reply:
👍